6. Developer Guides/09. Realtime-API/4. The Room Object/README.md

```diff
@@ -0,0 +1,84 @@
+# The Room Object
+
+The room object represents any form of communication between two or more users. There are at least two fields on the room object.
+
+- `_id`: The room / user id (depends on the room type)
+- `t`: The room type
+
+## Room Types
+
+- `d`: Direct chat
+- `c`: Chat
+- `p`: Private chat
+
+The information that comes with the room object changes according to its type.
+
+### Direct chat
+
+A direct chat between two users. In this case the room obejct only has two fields:
+
+- `_id`: The user id
+- `t`: The room type (in this case `d`)
+
+Example:
+
+```json
+{
+    "_id": "user-id",
+    "t": "d"
+}
+```
+
+You'll need to access the user information (using its id) in order to exhibit an name for this room.
+
+### Chat
+
+An open chat room. An chat contains more information about the room as it follows:
+
+- `_id`: The room id
+- `t`: The room type (in this case `c`)
+- `name`: The room name
+- `u`: The room creator (it may return a null user)
+- `topic`: (Optional) The room topic
+- `muted`: (Optional) A collection of muted users by its username
+- `jitsiTimeout`: (Optional) (?)
```

Comment:
@laggedHero its an ugly hack that needs to be fixed 😁  essentially client pings back from the jitsi panel while active...  and updates that jitsiTimeout so for people not connected we can use it to determine if session is active and indicate for them.